src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -67,56 +70,94 @@
     /** The end location */
     public GenericLocation to;
 
-    /** If true, the tree will be allowed to grow in all directions, rather than being directed toward a single target. */
+    /** If true, the tree will be allowed to grow in all directions, rather than being directed toward a single target.
+     * @Deprecated - Will not be supported in Raptor
+     * */
+    @Deprecated
     public boolean oneToMany = false;
 
-    /** An ordered list of intermediate locations to be visited. */
+    /** An ordered list of intermediate locations to be visited.
+     * @Deprecated - Not currently supported in Raptor, but planned in the future
+     * */
+    @Deprecated
     public List<GenericLocation> intermediatePlaces;
 
     /**
      * The maximum distance (in meters) the user is willing to walk for access/egress legs.
      * Defaults to unlimited.
+     *
+     * @Deprecated - TODO: This should be supported
      */
+    @Deprecated
     public double maxWalkDistance = Double.MAX_VALUE;
 
     /**
      * The maximum distance (in meters) the user is willing to walk for transfer legs.
      * Defaults to unlimited. Currently set to be the same value as maxWalkDistance.
+     *
+     * @Deprecated - TODO: This should be supported
      */
+    @Deprecated
     public double maxTransferWalkDistance = Double.MAX_VALUE;
 
     /**
      * The maximum time (in seconds) of pre-transit travel when using drive-to-transit (park and
      * ride or kiss and ride). By default limited to 30 minutes driving, because if it's unlimited on
      * large graphs the search becomes very slow.
+     *
+     * @Deprecated - Not currently supported in Raptor, but planned in the future
      */
+    @Deprecated
     public int maxPreTransitTime = 30 * 60;
 
-    /** The worst possible time (latest for depart-by and earliest for arrive-by) to accept */
+    /** The worst possible time (latest for depart-by and earliest for arrive-by) to accept
+     *
+     * @Deprecated - Not currently supported in Raptor, but planned in the future
+     * */
+    @Deprecated
     public long worstTime = Long.MAX_VALUE;
 
-    /** The worst possible weight that we will accept when planning a trip. */
+    /** The worst possible weight that we will accept when planning a trip.
+     *
+     * @Deprecated - Not supported in Raptor
+     * */
+    @Deprecated
     public double maxWeight = Double.MAX_VALUE;
 
-    /** The maximum duration of a returned itinerary, in hours. */
+    /** The maximum duration of a returned itinerary, in hours.
+     *
+     * @Deprecated - Not supported in Raptor
+     * */
+    @Deprecated
     public double maxHours = Double.MAX_VALUE;
 
-    /** Whether maxHours limit should consider wait/idle time between the itinerary and the requested arrive/depart time. */
+    /** Whether maxHours limit should consider wait/idle time between the itinerary and the requested arrive/depart time.
+     *
+     * @Deprecated - Not supported in Raptor
```

Comment:
I think some parameters you might consider very specific to narrow use cases were added as part of various projects by long-time OTP supporters like TriMet. This is related to the debates on e.g. whether the API should accept many different date formats. It may seem strange from an engineering perspective but it was convenient for certain users.